webclient/uzivatel/signals.py

```diff
@@ -34,9 +33,6 @@ def orgnaizace_save_metadata(sender, instance: Organizace, **kwargs):
 
 @receiver(pre_save, sender=Osoba, weak=False)
 def osoba_pre_save(sender, instance: Osoba, **kwargs):
-    if instance.orcid:
-        match = orcid_pattern.search(instance.orcid)
-        instance.orcid = match.group() if match else None
```

Comment:
@pesikj Tady moc nerozumím, proč zmizí validace pro ORCID, ale zůstane pro Wikidata. Navíc to celkem jistě nesouvisí s #2593, ale s #2594

Reply:
@motyc Pro Wikidata jsi můžu taky smazat.

Reply:
Nechám na Tobě, co Ti přijde lepší. Jen mě to zmátlo.